catchup/service.go

```diff
@@ -465,6 +465,10 @@ func (s *Service) periodicSync() {
 				sleepDuration = s.deadlineTimeout
 				continue
 			}
+			// if the catchup is disabled in the config file, just skip it.
```

Comment:
disabling it here still makes one catchup step via `s.sync()` above. Maybe exit earlier or not even run `periodicSync`?

Reply:
You're correct regarding the `s.sync()` above, but not running `periodicSync` completely is a problem since the agreement service depends on it ( i.e. it needs the `s.unmatchedPendingCertificates` case )